src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -598,18 +678,12 @@ function WorkspacesListPage() {
             <View style={styles.flex1}>
                 <TopBar breadcrumbLabel={translate('common.workspaces')}>{!shouldUseNarrowLayout && <View style={[styles.pr2]}>{getHeaderButton()}</View>}</TopBar>
                 {shouldUseNarrowLayout && <View style={[styles.ph5, styles.pt2]}>{getHeaderButton()}</View>}
-                <FlatList
-                    ref={flatlistRef}
-                    data={filteredWorkspaces}
-                    onScrollToIndexFailed={(info) => {
-                        flatlistRef.current?.scrollToOffset({
-                            offset: info.averageItemLength * info.index,
-                            animated: true,
-                        });
-                    }}
-                    renderItem={getMenuItem}
-                    ListHeaderComponent={listHeaderComponent}
-                    keyboardShouldPersistTaps="handled"
+                <SelectionList
+                    ref={listRef}
+                    sections={sections}
+                    sectionTitleStyles={[styles.ph5, styles.pb5, styles.mt0, styles.mb0, styles.pt3]}
+                    onSelectRow={() => {}}
```

Comment:
This is not the first time in the codebase when we pass an empty function to `onSelectRow`. After this gets merged it's gonna be 3/8 usages. I think that we can set `onSelectRow` as an optional prop for `SelectionList`